Makefile

```diff
@@ -120,14 +120,18 @@ acceptance:
 
 .PHONY: check
 check:
-	! git grep -F '"path"' -- '*.go'
-	! errcheck -ignore 'bytes:Write.*,io:(Close|Write),net:Close,net/http:(Close|Write),net/rpc:Close,os:Close,database/sql:Close,github.com/spf13/cobra:Usage' $(PKG) | grep -vE 'yacc\.go:'
-	! go-nyet $(PKG) | grep -vE '(Weird type of StarExpr|Unknown types|`matchIndex`|`c`|Illegal range|cannot process directory \.git|yacc\.go:)' # TODO(tamird): https://github.com/barakmich/go-nyet/pull/10
-	# https://golang.org/pkg/database/sql/driver/#Result :(
-	! golint $(PKG) | grep -vE '(\.pb\.go|embedded\.go|yyEofCode|_string\.go|LastInsertId|sql\.y|yacc\.go)'
-	! gofmt -s -l . | grep -vF 'No Exceptions'
-	! goimports -l . | grep -vF 'No Exceptions'
-	! go tool vet --shadow --shadowstrict . 2>&1 | grep -vE '(\.pb\.go|yacc\.go|declaration of err shadows|cannot process directory \.git)'
+	@echo "checking for \"path\" imports"
+	@! git grep -F '"path"' -- '*.go'
+	@echo "errcheck"
+	@! errcheck -ignore 'bytes:Write.*,io:(Close|Write),net:Close,net/http:(Close|Write),net/rpc:Close,os:Close,database/sql:Close,github.com/spf13/cobra:Usage' $(PKG) | grep -vE 'yacc\.go:'
+	@echo "vet"
+	@! go tool vet --shadow . 2>&1 | grep -vE '(\.pb\.go|yacc\.go|declaration of err shadows|cannot process directory \.git)'
+	@echo "golint"
+	@! golint $(PKG) | grep -vE '(\.pb\.go|embedded\.go|yyEofCode|_string\.go|LastInsertId|sql\.y|yacc\.go)' # https://golang.org/pkg/database/sql/driver/#Result :(
+	@echo "gofmt (simplify)"
+	@! gofmt -s -l . 2>&1 | grep -vE 'No Exceptions|\.go:1:1:'
```

Comment:
Ok, I rearranged this into two `grep -v` invocations I tightened up the check so that we just ignore errors in the `.git` directory.